builder/Dockerfile

```diff
@@ -1,14 +1,20 @@
-#This docker file is used in openshift CI
-FROM quay.io/kubevirt/common-templates:fedora33CI
+# This Dockerfile is used in openshift CI
+FROM quay.io/fedora/fedora:latest
```

Comment:
Can you use a fixed version instead of `latest`? Just to be sure that it is not unexpectedly updated.

Reply:
From what I've seen `latest` points to the latest Fedora `stable` release. Ideally, we would always use that anyway? Currently it is an additional maintenance task to update the image whenever a new Fedora `stable` is released.

Reply:
Ok